### PR TITLE
addded post list

### PR DESCRIPTION
**Pull request by shubham15-tech**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-// posts contract schema
-// [
-//   {
-//     id: randomID,
-//     title: "Post 1",
-//     content: "This is the first post",
-//     userId: 1,
-//   }
-// ];
+  const [posts, setPosts] = useState([
```
Comment (Rohitsonare) [inline src/App.js:6]: keep it emptyy

**Review by Rohitsonare — CHANGES_REQUESTED** [review]
small changes are required

**Review by Rohitsonare — APPROVED** [review]
nyc

File: src/App.js (hunk 1 above)
Comment (shubham15-tech) [reply src/App.js]: done thanks!

**Pull request merged**